capstone/src/main/java/com/step902020/capstone/OrganizationController.java

```diff
@@ -0,0 +1,66 @@
+
+package com.step902020.capstone;
+
+import java.io.IOException;
+import java.util.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.view.RedirectView;
+import com.google.gson.Gson;
+import java.io.IOException;
+
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@RestController
+public class OrganizationController {
+
+  @Autowired
+  private OrganizationRepository organizationRepository;
+
+  @GetMapping("get-organization")
+  public List<Organization> getOrganization(@RequestParam("email") String email) {
```

Comment:
Why would there ever be more than one organization with a specific email?
Also, I would probably prefer using `datastoreID` over email because an email of an organization can potentially be updated, but `datastoreID` is a better permanent primary key.

Reply:
There usually is just one entity returned. But sometimes none are found. Right now I handle this by passing the whole list and then checking for the list size on the front end. Is it better to do it this way or for me to pass just the Organization object, which may or may not be null.